classes/class.ilMumieTaskLPStatus.php

```diff
@@ -66,37 +66,59 @@ private static function updateResult($user_id, $taskId, $succeeded, $percentage)
      */
     public static function updateGrades($task, $force_update = false)
     {
-        include_once("Services/Tracking/classes/class.ilObjUserTracking.php");
+        if ($task->getPrivateGradepool() != -1) {
```

Comment:
Code get's easier to read if you just return e.g.

`if ($task->getPrivateGradepool() == -1) {
  return;
}`